actor/echo/README.md

```diff
@@ -0,0 +1,8 @@
+A simple http echo server
+This example demonstrates the actor code generation and
+a handler for the http-server interface.
+
+To build, `make` and `make run`
+
+To see the echo, curl a local url such as `curl localhost:8080/abc`
```

Comment:
Is this still accurate information? I don't see a `make run` rule, which may be a more complicated setup w/ the OTP runtime

Reply:
you're right. I'll remove the comment. We need to decide on how to document running actors, whether through the web ui or with a future manifest file.